tests/test_bad_dags.py

```diff
@@ -33,11 +33,16 @@
 import airflow_declarative
 
 
+text_type = type(u'')  # six.text_type
```

Comment:
Those things should be named in upper case as linters thinks these are global variables. But I'm not sure that we need in global variable just to not type `type(u'')` in one single case.

Reply:
The linting stage in Travis has passed.

I'd prefer to keep it a global variable, because this way the hack is more noticeable. Another reason is that the `six.text_type` comment might be placed there, which is:
a. greppable,
b. gives the exact idea of what is happening there and why.

Reply:
Ok, then please make it upper cased.  Linters (flake8) should be fixed on these cases. Pylint reports right about invalid name, but on those level which we cannot accept as fatal (that would be too hardcode).

Reply:
`flake8` tolerates this. `pylint` is not run against tests in our case.

I'd prefer to keep it as-is, because:
1. this way the name exactly matches the six's one,
2. there might be other lowercased module attributes, such as the module-level `pytestmark`,
3. `docs/conf.py` would have the same issue.

Reply:
Funny, but flake8 with pep8-naming doesn't handles this case (even derivative ones as they claim). Fine: no linter - no rule. Though, let's count this as exception: in the rest codebase we use upper case for global variables while `doc/conf.py` is a different story about Sphinx config API.